modules/ddl-and-loading/pages/defining-a-graph-schema.adoc

```diff
@@ -385,18 +385,20 @@ CREATE UNDIRECTED EDGE e (FROM v1, TO v2, att1 STRING COMPRESS e1)
 
 == `ALTER INDEX`
 
-User-defined indexes (or secondary indexes. as they are called commonly called in the database industry) are a valuable feature that enhances the performance of a database system. Indexes allow users to perform fast lookups on non-key columns or attributes without a full-fledged scan.
+User-defined indexes (or secondary indexes, as they are called commonly called in the database industry) are a valuable feature that enhances the performance of a database system. Indexes allow users to perform fast lookups on non-key columns or attributes without a full-fledged scan.
 
-The TigerGraph database allows users to define on vertex attributes. __**__User has the flexibility to create indexes in an empty graph initially or to add indexes later when the database is running. If the index is added on an existing vertex, index data is built in the background.
+The TigerGraph database allows users to define on vertex attributes.
```

Comment:
"to define on vertex attributes.":  This sounds odd because the object of "define" has been omitted.
Suggested change:
"to define indexes on vertex attributes."